### PR TITLE
Fix recurrency settings retrieval

### PR DESCRIPTION
**Pull request by vlaux**
#### What is the purpose of this pull request?
Fix the recurrency settings retrieval for saved promotions.

#### What problem is this solving?
Although the settings were being saved, the UI was incorrectly reading them, showing no recurrency settings after reload. 

#### How should this be manually tested?
Create a new promotion or edit an existing one adding custom recurrency settings. The settings should be preserved after refreshing page. 
https://vlauxexp--pricingqa.myvtex.com/admin/promotions/ 

#### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/utils/promotion/index.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ export const newPromotion = (intl, promotion, salesChannels) => {
   if (promotion) {
     const { generalInfo, eligibility, effects, restriction } = promotion
     const useRecurrency = !!generalInfo.cron
-    const [minute, hour, day, month, weekDay] = useRecurrency
+    const [, , hour, , , weekDay] = useRecurrency
```
Comment (darlenedms) [inline react/utils/promotion/index.js]: I think it's kinda hard to understand what are you trying to accomplish here, can you write it in a more explicit way or add a comment, please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+  - Recurrency settings retrieval for saved promotions
```
[the remark this answers is not in the document]
Comment (vlaux) [reply CHANGELOG.md]: My bad. Forgot to run it.

File: react/utils/promotion/index.js (hunk 1 above)
Comment (vlaux) [reply react/utils/promotion/index.js]: Done!
https://github.com/vtex/admin-promotions/pull/156/commits/1e8203a1ba7aef49b3239cad4585ee34048fd3d6

**Comment by ericreis** [comment]
I personally found the previous code simpler and easier to understand haha.

But if for everyone else this is better I have no problem with it 😬

**Pull request merged**